about/using-github.md

```diff
@@ -89,20 +89,22 @@ Remember that these repositories *are hosted* on GitHub servers, *NOT LLNL serve
 * NO Unclassified Controlled Information (UCI)
 * NO Unclassified Controlled Nuclear Information (UCNI)
 
-When in doubt, contact a Derivative Classifier (DC) and / or Information Management (IM) for further guidance.
```

Comment:
I think we need to keep IM spelled out, no?

I guess because we spell it out above?